src/components/Charts/ModelChart.js

```diff
@@ -288,8 +265,8 @@ const ModelChart = ({
         <Wrapper projections={projections} isInferred={projection.isInferred}>
           <Chart options={options} />
           <CondensedLegend>
-            {[noAction, socialDistancing, shelterInPlace, availableBeds]
-              .filter(intervention => intervention.visible !== false)
+            {[previousEstimates, noAction, projected]
+              .filter(series => series.visible !== false)
```

Comment:
socialDistancing is never shown in the charts, but we were still including it in the legend sometimes (see screenshot in PR description).